source/_docs/configuration/devices.markdown

```diff
@@ -12,7 +12,11 @@ Classification for the available integrations:
 - [IoT class](/blog/2016/02/12/classifying-the-internet-of-things): Classifier for the device's behavior.
 - [Quality scale](/docs/quality_scale/): Representation of the integration's quality.
 
-Usually every entity needs its own entry in the `configuration.yaml` file. There are two styles for multiple entries:
+Integrations that are automatically discovered, or manually added and configured via the UI, typically don't need an extra entry in the `configuration.yaml` file. Required parameters are directly entered during setup in the UI. However, please be aware that some UI integrations still might need additional configuration in the `configuration.yaml` file. These cases are explained in the respective integration documentation.
+
+Note, that for integrations configured in the UI, it is standard behavior that configuration parameters cannot be changed anymore. If the integration doesn't offer an UI option for change, delete the integration from the UI, let the system discover it again or add it back manually.
```

Comment:
This information is incorrect. You are describing an import method here, which is only used for deprecation periods (and instructions for these are provided via the release notes).

I suggest removing this section.

Reply:
Sorry. I'm only a user trying to contribute by making things clear that are totally confusing from a user perspective. As a user, I have no idea about what an import method is. And I shouldn't really have to. Point is: something apparently is standard behaviour that from a user perspective hides data that in the past (and for many integrations in the present) was visible and configurable. Deleting this paragraph would continue to leave everyone in the dard.

My documentation principle would be to always document current state. Not future desired state as this doesn't exist yet nor past state as this is irrelevant and confusing. Should the current state be one that is supposed to go away, than the documentation needs to evolve with the changing state. Which is what it needs to do anyway all the time.

Please accept the request, close it or modify it as you see fit.

Reply:
> As a user, I have no idea about what an import method is. And I shouldn't really have to.

That is correct, hence it doesn't belong here and should be removed.

> My documentation principle would be to always document current state. 

We have the same policy, and thus this is not a normal pattern, it should not be part of the docs.

Reply:
Sorry, I didn't mean to be rude. I guess I'm just lost.

As a user, I got stuck over a thing. You suggested to improve documentation, which is a fair request. I tried but I realize that I still haven't understood it well enough. Which makes the point that the current documentation doesn't work.

I'll close the PR.